src/PlatformAdapter/Native/Exec/NativeExecImpl.cpp

```diff
@@ -223,6 +216,15 @@ void NativeExecImpl::erase_task(NativeEvalTask* task)
   m_tasks.erase(task);
 }
 
+void NativeExecImpl::stop_node()
```

Comment:
better naming? should distinguish from `stop_node_thread_safe`

Reply:
updated